packages/keyboard-shortcuts/README.md

```diff
@@ -1,6 +1,6 @@
 # Keyboard Shortcuts
 
-Keyboard shortcuts is a generic package that allow registering andd modifying shortcuts.
+Keyboard shortcuts is a generic package that allow registering and modifying shortcuts.
```

Comment:
While we're fixing that, there is a small subject-verb agreement error here too we should fix. 
```suggestion
Keyboard shortcuts is a generic package that allows registering and modifying shortcuts.
```